contracts/call-number/tests/integration.rs

```diff
@@ -9,22 +9,22 @@ fn required_imports() -> Vec<(String, String, FunctionType)> {
     vec![
         (
             String::from("add"),
-            String::from("NumberContract"),
+            String::from("dynamiclinked_NumberContract"),
```

Comment:
If this string use many times, how about defining constant value?

Reply:
fixed and pushed. thanks!